src/style.scss

```diff
@@ -183,3 +183,10 @@
     padding-left: unset;
   }
 }
+
+:host([disabled]) {
+  .trigger {
+    pointer-events: none;
+    opacity: 0.5;
```

Comment:
We don't want to use opacity like this here. Instead we should be setting the text and chevron color to use the disabled token variant.

```
--ds-color-text-ui-disabled-default
--ds-color-icon-ui-disabled-default
```

Reply:
updated to using a token reference.